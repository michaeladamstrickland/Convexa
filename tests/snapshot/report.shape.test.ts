import { describe, it, expect } from 'vitest';

describe('report.json shape (skeleton)', () => {
  it('matches minimal shape snapshot', () => {
    const minimal = { totals: { total: 0, done: 0 }, costs: { total: 0 } };
    expect(minimal).toMatchSnapshot();
  });
});
