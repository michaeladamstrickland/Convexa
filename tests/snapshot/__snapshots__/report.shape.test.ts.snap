// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`report.json shape (skeleton) > matches minimal shape snapshot 1`] = `
{
  "costs": {
    "total": 0,
  },
  "totals": {
    "done": 0,
    "total": 0,
  },
}
`;
